vendor/github.com/elastic/gosigar/sigar_windows.go

```diff
@@ -408,3 +408,7 @@ func getWin32OperatingSystem() (Win32_OperatingSystem, error) {
 	}
 	return dst[0], nil
 }
+
+func GetRusage() (windows.Rusage, error) {
+	return nil, ErrNotImplemented{runtime.GOOS}
+}
```

Comment:
for windows have a look at PgSQL getrusage compat code: https://doxygen.postgresql.org/getrusage_8c.html